src/Portal/resources/dash1.json

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
If this file is used in test only, it may be better to put it into the test folder.